app/Schema/Sqlite.php

```diff
@@ -8,7 +8,52 @@
 use Kanboard\Core\Security\Role;
 use PDO;
 
-const VERSION = 124;
+const VERSION = 125;
+
+function version_125(PDO $pdo)
+{
+    $pdo->exec("
+      CREATE TABLE tasks_new
+        (
+            id                   INTEGER PRIMARY KEY,
+            title                TEXT NOCASE NOT NULL,
+            description          TEXT,
+            date_creation        INTEGER,
+            color_id             TEXT,
+            project_id           INTEGER REFERENCES projects ON DELETE CASCADE,
+            column_id            INTEGER REFERENCES columns ON DELETE CASCADE,
+            owner_id             INTEGER DEFAULT '0',
+            position             INTEGER,
+            is_active            INTEGER DEFAULT 1,
+            date_completed       INTEGER,
+            score                INTEGER,
+            date_due             INTEGER,
+            category_id          INTEGER DEFAULT 0,
+            creator_id           INTEGER DEFAULT '0',
+            date_modification    INTEGER DEFAULT '0',
+            reference            TEXT    DEFAULT '',
+            date_started         INTEGER,
+            time_spent           NUMERIC DEFAULT 0,
+            time_estimated       NUMERIC DEFAULT 0,
+            swimlane_id          INTEGER REFERENCES swimlanes ON DELETE CASCADE,
```

Comment:
It would be better to make the foreign column explicit, for example: `REFERENCES swimlanes(id)` (same for `projects` and `columns` tables above).

Reply:
done